### PR TITLE
Merge branch 'master' into cli-use-open-api and resolve conflicts

### PR DESCRIPTION
**Pull request by crelax**
### What problem does this PR solve?

Issue Number: close #xxx

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdc/api/v1/validator.go [hunk 1]
```diff
@@ -39,10 +39,12 @@ import (
 func VerifyCreateChangefeedConfig(
 	ctx context.Context,
 	changefeedConfig model.ChangefeedConfig,
-	capture capture.InfoForAPI,
+	capture capture.Capture,
 ) (*model.ChangeFeedInfo, error) {
 	// TODO(dongmen): we should pass ClusterID in ChangefeedConfig in the upcoming future
 	up := capture.GetUpstreamManager().GetDefaultUpstream()
+	//up := capture.GetUpstreamManager().Get(upstream.DefaultUpstreamID)
```
Comment (sdojjy) [inline cdc/api/v1/validator.go:46]: remove these two lines

File: cdc/capture/info_for_api.go [hunk 2]
```diff
@@ -13,23 +13,14 @@
...
-	GetOwner() (owner.Owner, error)
-	GetOwnerCaptureInfo(ctx context.Context) (*model.CaptureInfo, error)
-	StatusProvider() owner.StatusProvider
-	GetEtcdClient() etcd.CDCEtcdClientForAPI
-	GetUpstreamManager() *upstream.Manager
-}
+//// InfoForAPI provides capture's info and getters used for api
+//type InfoForAPI interface {
+//	Info() (model.CaptureInfo, error)
+//	IsOwner() bool
```
Comment (sdojjy) [inline cdc/capture/info_for_api.go:19]: remove

File: cdc/processor/processor_test.go [hunk 3]
```diff
@@ -94,9 +97,14 @@ func initProcessor4Test(ctx cdcContext.Context, t *testing.T) (*processor, *orch
     "sync-point-interval": 600000000000
 }
 `
+<<<<<<< HEAD
```
Comment (sdojjy) [inline cdc/processor/processor_test.go:100]: resolve conflict

**Review by sdojjy — APPROVED** [review]

**Review by zhaoxinyu — APPROVED** [review]

**Comment by asddongmen** [comment]
/ LGTM

**Review by asddongmen — APPROVED** [review]

**Pull request merged**